src/test/cypress/integration/exercises/text/TextExerciseAssessment.spec.ts

```diff
@@ -47,17 +47,13 @@ describe('Text exercise assessment', () => {
 
     it('Assesses the text exercise submission', () => {
         cy.login(tutor, '/course-management');
-        coursesPage.openAssessmentDashboardOfCourseWithId(course.id);
+        coursesPage.openAssessmentDashboardOfCourse(course.shortName);
         courseAssessment.checkShowFinishedExercises();
         courseAssessment.clickExerciseDashboardButton();
         exerciseAssessment.clickHaveReadInstructionsButton();
         cy.contains('There are no complaints at the moment').should('be.visible');
         cy.contains('There are no requests at the moment.').should('be.visible');
         exerciseAssessment.clickStartNewAssessment();
-        textAssessment.getInstructionsRootElement().contains(exercise.title).should('be.visible');
-        textAssessment.getInstructionsRootElement().contains(exercise.problemStatement).should('be.visible');
-        textAssessment.getInstructionsRootElement().contains(exercise.sampleSolution).should('be.visible');
-        textAssessment.getInstructionsRootElement().contains(exercise.gradingInstructions).should('be.visible');
```

Comment:
Are these assertions not relevant anymore?

Reply:
The question is if they ever were relevant. To the test they are not, but I guess they still make sense to verify the view for the tutor, so I will readd them